templates/products/products-listing-row.php

```diff
@@ -139,13 +139,13 @@
             if ( $time_diff > 0 && $time_diff < 24 * 60 * 60 ) {
                 $h_time = sprintf( __( '%s ago', 'dokan-lite' ), human_time_diff( $time ) );
             } else {
-                $h_time = mysql2date( __( 'Y/m/d', 'dokan-lite' ), $m_time );
+                $h_time = $m_time;
```

Comment:
here: use dokan_date_time_format($m_time)

Reply:
On line 134 use $m_time = $post->post_date_gmt;

Reply:
on second thought On line 134 use $m_time = $post->post_date; is enough, I've checked dokan_date_time_format(), this doesn't double convert time.

Reply:
Bhai, I've refactored the whole block for better understanding and used `get_post_time( 'c', true, $post );` for formating.